package.json

```diff
@@ -24,13 +24,19 @@
     "author": "Kumulos Ltd. <support@kumulos.com>",
     "license": "MIT",
     "devDependencies": {
+        "css-loader": "^3.4.0",
+        "node-sass": "^4.13.0",
         "prettier": "^1.18.2",
+        "sass-loader": "^8.0.0",
+        "style-loader": "^1.0.2",
         "ts-loader": "^6.2.0",
         "typescript": "^3.7.2",
         "uglify-es": "^3.3.9",
         "uglifyjs-webpack-plugin": "^2.2.0",
         "webpack": "^4.41.2",
         "webpack-cli": "^3.3.9"
     },
-    "dependencies": {}
+    "dependencies": {
```

Comment:
Should this be a peer or dev dependency? Technically you'd need it if building from source, but you shouldn't need it to consume the dist files (default).

Reply:
OK after some reading, seems like keeping as a dependency might be the best choice:  https://lexi-lambda.github.io/blog/2016/08/24/understanding-the-npm-dependency-model/